sdk/Tests/Unit/Trace/TracingTest.php

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
We were working on leaving the tests in the `tests` directory - was this an inadvertent move?  I spoke to this more in our gitter channel.

Reply:
done. I moved all tests inside the same root directory